### ISSUE TITLE
Exception error when requesting current or single update in example app?

### ISSUE THREAD

**Issue by Carrein** [loup-v/geolocation#20]
I ran the example application provided by the repo and while the first and all subsequent 'Last Known' location works - clicking 'Current' or 'Single Update' only shows 'In progress...'

The error log is as follows:

```
I/flutter ( 9240): ══╡ EXCEPTION CAUGHT BY GEOLOCATION ╞═══════════════════════════════════════════════════════════════
I/flutter ( 9240): The following PlatformException was thrown while invoking
I/flutter ( 9240): geolocation/location/addLocationUpdatesRequest:
I/flutter ( 9240): PlatformException(error, Expected BEGIN_OBJECT but was STRING at path $.options, null)
I/flutter ( 9240):
I/flutter ( 9240): When the exception was thrown, this was the stack:
I/flutter ( 9240): #0      StandardMethodCodec.decodeEnvelope (package:flutter/src/services/message_codecs.dart:547:7)
I/flutter ( 9240): #1      MethodChannel.invokeMethod (package:flutter/src/services/platform_channel.dart:279:18)
I/flutter ( 9240): <asynchronous suspension>
I/flutter ( 9240): #2      _invokeChannelMethod (file:///C:/Users/addis/Downloads/geolocation-master/geolocation-master/lib/channel/helper.dart:12:26)
I/flutter ( 9240): <asynchronous suspension>
I/flutter ( 9240): #3      _LocationChannel.locationUpdates.<anonymous closure> (file:///C:/Users/addis/Downloads/geolocation-master/geolocation-master/lib/channel/location_channel.dart:101:9)
I/flutter ( 9240): #8      _TabLocationState._listenToLocation (file:///C:/Users/addis/Downloads/geolocation-master/geolocation-master/example/lib/tab_location.dart:49:33)
I/flutter ( 9240): #9      _TabLocationState._onCurrentPressed (file:///C:/Users/addis/Downloads/geolocation-master/geolocation-master/example/lib/tab_location.dart:38:5)
I/flutter ( 9240): #10     GestureRecognizer.invokeCallback (package:flutter/src/gestures/recognizer.dart:102:24)
I/flutter ( 9240): #11     TapGestureRecognizer._checkUp (package:flutter/src/gestures/tap.dart:161:9)
I/flutter ( 9240): #12     TapGestureRecognizer.acceptGesture (package:flutter/src/gestures/tap.dart:123:7)
I/flutter ( 9240): #13     GestureArenaManager.sweep (package:flutter/src/gestures/arena.dart:156:27)
I/flutter ( 9240): #14     _WidgetsFlutterBinding&BindingBase&GestureBinding.handleEvent (package:flutter/src/gestures/binding.dart:147:20)
I/flutter ( 9240): #15     _WidgetsFlutterBinding&BindingBase&GestureBinding.dispatchEvent (package:flutter/src/gestures/binding.dart:121:22)
I/flutter ( 9240): #16     _WidgetsFlutterBinding&BindingBase&GestureBinding._handlePointerEvent (package:flutter/src/gestures/binding.dart:101:7)
I/flutter ( 9240): #17     _WidgetsFlutterBinding&BindingBase&GestureBinding._flushPointerEventQueue (package:flutter/src/gestures/binding.dart:64:7)
I/flutter ( 9240): #18     _WidgetsFlutterBinding&BindingBase&GestureBinding._handlePointerDataPacket (package:flutter/src/gestures/binding.dart:48:7)
I/flutter ( 9240): #19     _invoke1 (dart:ui/hooks.dart:134:13)
I/flutter ( 9240): #20     _dispatchPointerDataPacket (dart:ui/hooks.dart:91:5)
I/flutter ( 9240): (elided 4 frames from package dart:async)
I/flutter ( 9240): ════════════════════════════════════════════════════════════════════════════════════════════════════
I/flutter ( 9240): location result: null
```

The app was run on the default AVD Nexus 5X device on Android 8.1.

**Comment by shehabic-work** [comment]
@lukaspili is this correct: 
https://github.com/loup-v/geolocation/commit/60f0d5925fbd51b9168fdbf082874c411f6b9be9#diff-a8c3e9ee260fd6f677546fc50a362fccR109 ?
It's not an enum and it sends the param as: `Instance of AndroidOptions`

**Comment by lukaspili** [comment]
Sorry for the delay. It's fixed in the latest release 1.0.0.
